akamai.services.yml

```diff
@@ -0,0 +1,8 @@
+services:
+  akamai.akamaiservice:
+    class: Drupal\akamai\AkamaiContentControlClient
+    arguments: ['@config.factory', '@logger.channel.akamai']
+  logger.channel.akamai:
```

Comment:
Apparently the channel is the the equivalent of calling `watchdog($type)` - where the channel == type. Then, in your module you can just call the logger and everything will be categorised under that channel name.

I based this on the logger in Aggregator.

https://www.drupal.org/node/2270941 - change notice where `watchdog` was removed